webapp/dispatch.yaml

```diff
@@ -0,0 +1,3 @@
+dispatch:
+  - url: "*/api/revisions/*"
```

Comment:
There is an unlikely chance that we would ship `/something/else/api/revisions/*` in AppEngine Standard, and it wouldn't work, but I doubt it. We could have something like `*wpt.fyi/...` AND `*appspot.com/...` if we want to guard against that.